tests/periph_uart/tests/periph_uart_if.py

```diff
@@ -6,11 +6,15 @@
 """@package PyToAPI
 This module handles parsing of information from RIOT periph_uart test.
 """
+import logging
+
 from riot_pal import DutShell
 
 
 class PeriphUartIf(DutShell):
     """Interface to the node with periph_uart firmware."""
+
+
```

Comment:
I am probably wrong but shouldn't one newline make the linter happy?

Reply:
yeah might be, I think I copied/adapted from the `periph_i2c_if.py` but I'll check and amend a proper version.